File: NetFramework/TouchToolkit/VB/TouchToolkitDemo/GcZoomPages/Help Forms/InputFormforControlBar.vb
Imports System.Collections.Generic
Imports System.ComponentModel
Imports System.Data
Imports System.Drawing
Imports System.Linq
Imports System.Text
Imports System.Windows.Forms

Namespace C1ZoomPages
	Public Partial Class InputFormforControlBar
		Inherits Form
		Public Sub New()
			InitializeComponent()
		End Sub

		Protected Overrides Sub OnLoad(e As EventArgs)
			MyBase.OnLoad(e)
			Me.gcZoom1.ControlBars(0).Position = New Point(Me.Width, 0)
		End Sub
	End Class
End Namespace

File: NetFramework/WinForms/CS/TouchDemo/ZoomPages/Help Forms/InputFormforControlBar.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Linq;
using System.Text;
using System.Windows.Forms;

namespace TouchDemo.C1ZoomPages
{
    public partial class InputFormforControlBar : Form
    {
        public InputFormforControlBar()
        {
            InitializeComponent();
        }

        protected override void OnLoad(EventArgs e)
        {
            base.OnLoad(e);
            this.C1Zoom1.ControlBars[0].Position = new Point(this.Width ,0);
        }
    }
}
